wgpu/src/lib.rs

```diff
@@ -227,6 +228,14 @@ pub struct Texture {
     id: ObjectId,
     data: Box<Data>,
     owned: bool,
+    width: u32,
+    height: u32,
+    depth_or_array_layers: u32,
+    mip_level_count: u32,
+    sample_count: u32,
+    dimension: TextureDimension,
+    format: TextureFormat,
+    usage: TextureUsages,
```

Comment:
Should we just store the descriptor?

Reply:
This is possible, but the lifetime of the label might be an issue. I can just set it to None.